docs/community/com-query/grafana-connector.md

```diff
@@ -51,26 +58,26 @@ From `grafana-7-x.x/` run the following command:
 ### Add Datasource
 
 - Sign in or create credentials (you can use admin/admin if logging in for the first time)
-- Add Molecula as a datasource in Grafana and enter FeatureBase server information
+- Add Featurebase as a datasource in Grafana and enter FeatureBase server information
     - The welcome page should include a link to the [Add data source page](http://localhost:3000/datasources).
       - See the [Grafana docs](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source) for more details.
-    - If your molecula-datasource plugin is installed, it should appear in the data source list.
-
+    - If your featurebase-datasource plugin is installed, it should appear in the data source list.
 
 ![Grafana Welcome](/assets/images/grafana-welcome.png "Grafana Welcome")  
 *Click "Add your first data source"*
```

Comment:
Why is this set as italics with quotation marks?
When does the user click "Add your first data source"?

Instructions should be:

* bullets
* bold only the button
* no italics

For example:

```
* Click **Add your first data source**
```

A related question -- what's the button called when the user wants to add *another* data source?